### PR TITLE
Allow for a few 404s on status polling for cases where they're not in…

### PR DESCRIPTION
**Pull request by geoffjentry**
… metadata yet

### CODE REVIEW

**Comment by geoffjentry** [comment]
This feels kinda yucky to me but it works

File: src/main/scala/centaur/test/Test.scala [hunk 1]
```diff
@@ -80,17 +80,22 @@ object Operations {
     * stops with a failure.
     */
   def pollUntilStatus(workflow: SubmittedWorkflow, expectedStatus: WorkflowStatus): Test[SubmittedWorkflow] = {
+    def pollDelay() = Thread.sleep(10000) // This could be a lot smarter, including cromwell style backoff
```
Comment (mcovarr) [inline src/main/scala/centaur/test/Test.scala:83]: This is quite generous, did you find it actually needed to be so large?

Comment (geoffjentry) [inline src/main/scala/centaur/test/Test.scala:83]: No. I won't even pretend like the number 2 had any sort of logical basis. At least I know where the 10 seconds came from and that was purely using the other one. From what I saw it was usually < 1s TBH

**Comment by mcovarr** [comment]
:+1: eventual consistency whadda ya gonna do

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/87/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Comment by Horneth** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/87/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Pull request merged**